keyboards/lets_split/keymaps/tylerwince/keymap.c

```diff
@@ -0,0 +1,233 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+enum layer_names {
+  _QWERTY = 0,
+  _COLEMAK,
+  _DVORAK,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+};
+
```

Comment:
Your `custom_keycodes` code block was actually fine; it was only the layer names that needed adjusting.

Everything after your Tap Dance declarations should be restored to how it was at https://github.com/qmk/qmk_firmware/blob/d8ba849efb4d24e3f80c2eb548c7a0086455589f/keyboards/lets_split/keymaps/tylerwince/keymap.c, from here to the end of the file should match Lines 29-249 of the linked revision of `keymap.c`.

Reply:
Ahh - understood. I’ll make the change when I am stationary and push.